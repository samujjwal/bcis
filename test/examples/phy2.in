unit meter
unit second
unit velocity = meter/second
unit testunit = meter*second
set x = 1* velocity *(1* velocity)
set y = 1* velocity ^ 2
set z = 1*second^2
set w = 1*testunit^3

print(x)
print(y)
print(z)
print(w)
