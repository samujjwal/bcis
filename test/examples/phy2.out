1*meter^2/second^2
1*meter^2/second^2
1*second^2
1*meter^3/second^3
